src/components/settings/Account/Billing/Pricing.tsx

```diff
@@ -1,31 +1,29 @@
 import { SimpleGrid } from "@chakra-ui/react";
-import { AccountPlan } from "@3rdweb-sdk/react/hooks/useApi";
+import { Account, AccountPlan } from "@3rdweb-sdk/react/hooks/useApi";
 import { PricingCard } from "components/homepage/sections/PricingCard";
 import { useMemo } from "react";
 import { CONTACT_US_URL } from "utils/pricing";
 import { remainingDays } from "utils/date-utils";
 
 interface BillingPricingProps {
-  plan: AccountPlan;
+  account: Account;
```

Comment:
the word `Account` means a specific thing in our `v5` sdk and I don't love that we're overloading this here, better would be `ThirdwebAccount` or something here

Reply:
You're right, we probably shouldn't pass in the whole account object if its not needed (only 1-2 properites), reverted ✅